FINO_20100513to26/templates/namelist.input.template_NARR

```diff
@@ -38,7 +38,7 @@
 /
 
 &domains
- time_step                 =  90,
+ time_step                 =  75,
```

Comment:
Should `dx` and `dy` below also be updated to match the other namelist input files?

Reply:
Fixed in #19